Cargo.lock

```diff
@@ -4974,3 +5145,29 @@ dependencies = [
  "syn 1.0.65",
  "synstructure",
 ]
+
+[[package]]
+name = "zeromq"
+version = "0.3.2"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "66b9ebf6a24cc77c2c7053f8f0df07e6261abc5166c060d874efc828080c7316"
+dependencies = [
+ "async-trait",
+ "asynchronous-codec",
+ "bytes",
+ "crossbeam",
+ "dashmap 3.11.10",
```

Comment:
Multiple versions of this special hashmap optimized for parallel access in our single threaded program. Sigh.